### ISSUE TITLE
Create federated version of the "discover" view

### ISSUE THREAD

**Issue by mouse-reeve** [bookwyrm-social/bookwyrm#1473]
This will entail re-adding the federated activitystream code (https://github.com/bookwyrm-social/bookwyrm/pull/1253/commits/75f586c10983e70169df495b835d4ffa58757454), and adding a federated tab to the discover view. I think there should be brief, explanatory text about what "local" and "federated" mean

**Comment by mojosam** [comment]
I support this idea. (I don't know how GitHub works. I don't know if I'm allowed to say that. How do we know which are the popular ideas?)

**Comment by mouse-reeve** [comment]
thank you! a comment or thumbs up emoji react are both great :)

**Comment by mouse-reeve** [comment]
Closing this as a duplicate of #2751 and the conversation here: https://decisions.joinbookwyrm.com/d/TJ8QMLbR/re-thinking-the-discover-timeline

**Comment by beedell-roke** [comment]
@mouse-reeve, GitHub, nowadays, allows you to [re-]close as a duplicate, wherein you can enter the URI of the issue that this one duplicates.

**Comment by mouse-reeve** [comment]
You're right, I completely gloss over the fact that that button isn't just one thing. Thanks!
